server/Comments/comment.datastore.js

```diff
@@ -23,7 +23,7 @@ const commentDatastore = {
     },
 
     getAllComments: (id, success, fail) => {
-        Comment.find({"postId": id})
+        Comment.find({"postId": id}).populate('userId')
```

Comment:
Vulnerability! Will populate with username and password fields! Filter out anything you do not need.

Reply:
For example, if only a user's fullName is needed:
`.populate({path: 'likedPosts', populate: {path: 'userId', select: 'fullName'}})`